script/test

```diff
@@ -381,6 +392,9 @@ main()
                 shift
                 do_cert_suite "$@"
                 ;;
+            cert_unit_tests)
```

Comment:
I see certification tests more as functional tests - they involve much more than what a unit test should cover.

Can we call this `unit_tests` instead?